commit f99f6af48fcc27d219a6c5e344712dd67d12a7c7
Author: Ake Sandgren <ake.sandgren@hpc2n.umu.se>
Date:   Tue Mar 16 13:05:49 2021 +0100

    Rely on mpi compiler wrappers to handle include/lib for MPI.

diff --git a/src/Makefile b/src/Makefile
index b1db244..83ee3d7 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -51,8 +51,7 @@ NVLDFLAGS += -L$(NCCL_HOME)/lib
 endif
 
 ifeq ($(MPI), 1)
-NVCUFLAGS += -DMPI_SUPPORT -I$(MPI_HOME)/include
-NVLDFLAGS += -L$(MPI_HOME)/lib -lmpi
+NVCUFLAGS += -DMPI_SUPPORT
 endif
 LIBRARIES += curand nccl nvToolsExt
 NVLDFLAGS += $(LIBRARIES:%=-l%)
